### PR TITLE
add-new-language-Navajo

### PR DESCRIPTION
**Pull request by SamiraMSadat**
Includes the following tickets:
473 - Original Site, 404 Page, WY Link Text, New Footer, Election Dates
458 - Accordion component (all sections and revisions)
493 - About Us Page
503 - ARIA tags

Resolves Issue #476

### CODE REVIEW

File: content/nv/about-us.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
++++
+title = "About Vote.gov"
```
Comment (rayestrada) [inline content/nv/about-us.html:2]: The `title` needs to be translated.

File: data/translations/nv/accordion.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "sub_head_part_4": "T0nteel ts’7naag00 d7n7n1ago",
+    "content_6_part_4": "U.S. biyi’d00 dine’4 n7l47 t0nteel bil1ahdi nijin7n1ago 47 a’da’ii’n77[ biniy4 hazhdi’dool77[ d00 bee ada’ii’n7[7 [a’ j0ki’ 47 naaltsoos the %link%. ",
+    "content_6_part_4_link": "Federal Post Card Application (FPCA)",
+    "content_6_part_5": "woly4ego hach’8’ 1l’88h. Kwe’4 the %link% woly4ego d7n77[’88[ 47 hazh0’0 yee ni[ hodoolnih sil1otahdi nijigh1ago d00 t0nteel bil11hd66’ a’ji’1adgo.",
+    "content_6_part_5_link": "Federal Voting Assistance Program (FVAP)"
+}
```
Comment (rayestrada) [inline data/translations/nv/accordion.json:45]: This file is no longer required. You have to migrate this content into the `/content/<language-code>/accordion` directory as the others are.

File: data/translations/nv/time.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    "9:45 PM": "오후 9시 45분",
+    "10:00 PM": "오후 10시",
+    "10:15 PM": "오후 10시 15분",
+    "10:30 PM": "오후 10시 30분",
+    "10:45 PM": "오후 10시 45분",
+    "11:00 PM": "오후 11시",
+    "11:15 PM": "오후 11시 15분",
+    "11:30 PM": "오후 11시 30분",
+    "11:45 PM": "오후 11시 45분"
+}
```
Comment (rayestrada) [inline data/translations/nv/time.json:98]: These times are Korean. Replace with navajo time translations? or if we do not have them use english times.

**Review by rayestrada — CHANGES_REQUESTED** [review]

**Comment by rayestrada** [comment]
In addition we need a localized date for navajo in `/layouts/partials/date.html`

File: data/translations/nv/time.json (hunk 3 above)
Comment (SamiraMSadat) [reply data/translations/nv/time.json:98]: I copied this file over from English. Looks like that one is wrong too. I'll change both to English (I have no idea why English is displaying correctly).
We are using English until we get time back from the translator.

File: data/translations/nv/accordion.json (hunk 2 above)
Comment (SamiraMSadat) [reply data/translations/nv/accordion.json:45]: How (and from where) are these .md files generated?

File: assets/fonts/navajo-font/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  font-display: swap;
+}
+
+*,
+*.usa-button,
+*.main-heading,
+*.content-heading,
+*.usa-label,
+*.usa-input,
+*.usa-select[data-test="state-selection"],
```
Comment (rayestrada) [inline assets/fonts/navajo-font/stylesheet.css:45]: @SamiraMSadat add
```
*.usa-prose>h2,
*.usa-prose>h3,
*.usa-prose>h4,
*.usa-prose>h5,
```

**Review by rayestrada — APPROVED** [review]

**Pull request merged**